domains/fawwazalfarisi.json

```
@@ -4,7 +4,8 @@
     "email": "fawwazalfarisi.works@gmail.com"
   },
   "records": {
-    "CNAME": "fawwaz-dev.vercel.app"
+    "CNAME": "fawwaz-dev.vercel.app",
```

```suggestion
    "CNAME": "fawwaz-dev.vercel.app"
```
Invalid Records, please read -> [Record Types](https://docs.is-a.dev/domain-structure/#record-type-examples)
Please read our Vercel guide -> https://docs.is-a.dev/guides/vercel/